StripePayments/StripePayments/Source/API Bindings/Models/STPElementsSession.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  STPElementsSession.swift
+//  StripePayments
+//
+//  Created by Nick Porter on 2/15/23.
+//
+
+import Foundation
+
+public class STPElementsSession: NSObject {
+
+    /// The list of payment method types (e.g. `[NSNumber(value: STPPaymentMethodType.card.rawValue)]`) that this ElementsSession is allowed to use.
+    @objc public let paymentMethodTypes: [NSNumber]
```

Comment:
`@_spi(STP)`?

Reply:
Yeah, although now that the entire class is under SPI I'm not sure we need the property level ones.

Reply:
I'd guess you don't need property level ones anymore.